pkgs/applications/editors/emacs/elisp-packages/libgenerated.nix

```diff
@@ -1,6 +1,9 @@
 lib: self:
 
 let
+  inherit (lib) elemAt;
```

Comment:
nit: the style of this file is to explicitly use `lib.foo`.

Reply:
I'd rather refactor the rest of them to use the `inherit` style.
It's both more readable and more performant.